src/store/getters/state-getters.ts

```diff
@@ -533,8 +534,16 @@ export const isIncorporationApplicationValid = (state: StateIF): boolean => {
   )
 }
 
+/** Whether the partial registration steps are valid. */
+export const isPartialRegistrationValid = (state: StateIF): boolean => {
+  return (
+    getRegistration(state).defineBusinessValid &&
+    isAddPeopleAndRolesValid(state)
+  )
+}
+
 /** Whether all the registration steps are valid. */
-export const isRegistrationValid = (state: StateIF): boolean => {
+export const isFinalRegistrationValid = (state: StateIF): boolean => {
```

Comment:
I'm not sure about the name change here. 
This computed value is still the registration validity as a whole?

Reply:
The name change is to separate the partial registration validation from the final registration validation. I can change it back. Yes, this computed value is still the registration validity as a whole. If it is not a whole, then something bad might happen. Like if the user visits the final stepper and checks everything, returns to the previous step to remove something, then comes back the final stepper, it will still be valid.